Week2/LanguageTrigger.sql

```diff
@@ -0,0 +1,40 @@
+
+-- Gets alerts when a country has >= 10 languages.
+DROP TRIGGER IF EXISTS `new_world`.`countrylanguage_AFTER_INSERT`;
```

Comment:
your trigger is good I would just have done  it on a BEFORE than on AFTER. Better to raise error as soon as possible.